css/headline-section.css

```diff
@@ -0,0 +1,39 @@
+.illustration-head-line {
+  background-image: url("../img/illustration-head-line.png");
+  background-position-y: 60px;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.container-Head-Line {
+  margin-left: 15px;
+  margin-right: 15px;
+  padding-top: 30px;
+  padding-bottom: 40px;
+}

```

Comment:
- According to the project requirements, this section should be aligned using the `flex` display property, please fix this 🙏 🙏